docs/api-versioning.md

```diff
@@ -0,0 +1,103 @@
+# API Versioning for Dicom Server
+
+This guide gives an overview of the API versioning of the REST endpoints for Dicom Server.
+
+## Routes
+
+API Version number are set within the route. Example:
+`/v1.0-prerelease/studies`
+
+To add a route, use the `[VersionedRoute]` attribute to automatically add the version number to the route. Example:
```

Comment:
Eg. From a consumer's perspective we should list what sort of changes are will be included in a new version, while from the developer side this may be a checklist of what is a "breaking change" so they can be made aware while coding

Reply:
I had initially thought of this as only a developer document, and that the customer docs would be separate and live on a docs.microsoft.com page or something. But I guess this is public and therefore also visible to customers. 

I can make this more clear that this is developer guidance, and add separate info for customers